tests/manager/openAccount/managerCanOpenAccount.spec.js

```
@@ -28,4 +56,13 @@ test('Assert manager can add new customer', async ({ page }) => {
   1. Do not rely on the customer row id for the step 13. 
     Use the ".last()" locator to get the last row.
   */
+
+  await bankManagerMainPage.clickAccountLink()
+  await accountPage.selectCustomer(firstName +' '+lastName)
+  await accountPage.selectCurrency('Dollar')
+  await accountPage.clickProcessButton()
+  await page.reload()
```

Using `await page.reload()` after `clickProcessButton()` to close the success popup is brittle. Instead, accept the alert (if the app shows a browser alert) or wait for a success UI change. Relying on reload can mask failures and lead to timing issues.